src/components/NavigationPopupMenu/NavigationPopupMenu.js

```diff
@@ -17,6 +17,17 @@ import useLockBodyScroll from './../../utils/useLockBodyScroll'
 
 import LoaderSpinner from './../LoaderSpinner'
 
+/**
+ * Component that renders the NavigationPopupMenu component.
+ *
+ * @component
+ *
+ * @param {} loggedIn - Parameter is not used within this component.
+ * @param {booelan} showBanner - Parameter that if set to true, will show the banner.
```

Comment:
Small spelling mistake

Reply:
I see indeed haha, I've fixed the spelling mistake in my last commit.